packages/target-decisioning-engine/src/index.js

```diff
@@ -13,7 +13,7 @@ import { GeoProvider } from "./geoProvider";
  * The TargetDecisioningEngine initialize method
  * @param {import("../types/DecisioningConfig").DecisioningConfig} config Options map, required
  */
-export default function TargetDecisioningEngine(config) {
+export default function TargetDecisioningEngine(config, telemetryProvider) {
```

Comment:
please update JSDOC for this function, and take a look at any other functions  where function signatures have changed like this and update them as well.